docs/content/python/getting-started/quickstart/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Quickstart (5 min)
+description: |
+  Ready to get started? See Oso in action, and walk through adding roles to an app.
+weight: 1
+---
+
+# Oso Quickstart
+
+Oso is an open-source, batteries-included library for authorizing actions in your app.
+Out of the box, Oso lets you give your users roles and lets you specify permissions for those roles.
+Roles can be as simple as "user" and "admin", or as complex as a management hierarchy.
+
+![Diagram showing an application hierarchy with site admins, store owners, and customers](/getting-started/quickstart/images/app-hierarchy.png)
+
+Oso isn't restricted to roles, though — you can replace any authorization code in your app with an Oso policy.
+
+Why use Oso? Authorization always starts out simple, but can be increasingly
+difficult to manage as your app grows. Oso's design will guide you to best practices.
+
+Oso is a library — it runs alongside your app code and doesn't make any calls over the network.
+Your data doesn't leave your server. Oso also doesn't persist user data inside the library, so you stay in control of your data.
+
+Here's how data flows between your app and the Oso library:
+
+![Architecture diagram for Oso library loading a policy file and making authorization decisions. ](/getting-started/quickstart/images/arch-simple.png)
+## Install the Oso library
+
+{{% exampleGet "installation_new" %}}
+
+## Add Oso to your app
```

Comment:
In their current place, these two sections feel like a bit of a jarring departure from the higher-level-focused prose on either side of them.

current flow is something like this:

- introduction
- why use Oso?
- what is Oso?
- instructions on installing Oso
- instructions on setting Oso up
- how does Oso work? (`Accept or deny requests`)

If the point we're trying to make is that it's super easy to install & set up Oso, then I think install + setup should go right after the introduction (just before the line that starts with `Why use Oso?`) and make it clear that we're highlighting how easy it is to get going.

If the install + setup sections are intended as steps to code along with for someone reading this guide, then we should add some encouraging CTAs (and we'll have to rewrite a good chunk of the guide / example to make it interactive, so I don't think this is the plan for right now.

If we don't care about people coding along with the guide for the time being, I would either condense these sections into a brief callout whose purpose is to show how easy it is to install & set up Oso or just get rid of them entirely since this guide currently seems to be about showing folks what they can do with Oso once it's already set up, and we could explain set up elsewhere.

Reply:
I think we could improve this for the time being with a short transition before the Installation.

Something like:

"In this guide, we'll show you how to use Oso in your application to accept or deny requests with role based access control."